sphinx/writers/latex.py

```diff
@@ -796,7 +796,7 @@ def generate_numfig_format(self, builder):
             ret.append(self.babel_renewcommand('\\figurename', definition))
             if figure[1]:
```

Comment:
the difficulty is that we don't renew `\fnum@figure` it will be `\figurename\nobreakspace\thefigure`. This might be the reason why `\figurename` was fetched from `numfig_format` via action of `strip()`. I think preferable to not `strip()` and always redefine `\fnum@figure` without the `\nobreakspace`.